src/pre_defined_classes/simple_programs_taking_user_input/hard/q1/PayrollTaxTest.java

```diff
@@ -0,0 +1,35 @@
+package pre_defined_classes.simple_programs_taking_user_input.hard.q1;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.RepeatedTest;
+
+import java.nio.channels.NonWritableChannelException;
+
+public class PayrollTaxTest extends BaseTest {
+
+    public Clause[] testSentence() {
+        provideInput("48.75 59.86 0.16 ");
```

Comment:
not sure if it works if you put it here, it should be in the run main method. Also if you're inputting multiple variables it has to be separated by System.lineSeparator()

Reply:
Actually it is working without that